scout/server/blueprints/variant/utils.py

```diff
@@ -474,6 +483,7 @@ def frequency(variant_obj):
         variant_obj.get("gnomad_frequency") or 0,
         variant_obj.get("gnomad_mt_homoplasmic_frequency") or 0,
         variant_obj.get("swegen_mei_max") or 0,
+        variant_obj.get("colorsdb_af") or 0,
```

Comment:
Nice, I think there will be many long-read variants not present in gnomad, but only in colors_db.